openapi.yaml

```diff
@@ -3492,12 +3502,12 @@ paths:
                     type: string
                     nullable: true
                     description: |-
-                      Name of the plan the user has subscribed to.
+                      Name of the plan the user has subscribed to, if any.
```

Comment:
"has subscribed to" -> "is currently subscribed to"

(it's not necessarily the same, e.g.  expiring plans)

Reply:
```suggestion
                      Name of the plan the user is currently subscribed to if any.
```

Reply:
Fixed